vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -39977,3 +39985,9459 @@ unit:PERCENT-PER-MO qudt:symbol "%/mo" .
 # found by the algorithm but generated wrong. Here is my manual attempt:
 unit:P qudt:symbol "p" .
 unit:PT_BIG qudt:symbol "pb" .
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+######################################################################################################
+##
+##         UCUM CODES
+##
+######################################################################################################
+  # MISSING UCUMCODE: unit:AT - calculated from factors: A
+  #
+  #  ─── unit:AT ucumCode: [no ucumCode] (correct: A)
+unit:AT qudt:ucumCode "A"^^<http://qudt.org/schema/qudt/UCUMcs> .
+
+  # MISSING UCUMCODE: unit:A_Stat - calculated from factors: A
+  #
+  #  ──┬ unit:A_Stat ucumCode: [no ucumCode] (correct: A)
+  #    └─── unit:A ucumCode: A
+unit:A_Stat qudt:ucumCode "A"^^<http://qudt.org/schema/qudt/UCUMcs> .
+
+  # MISSING UCUMCODE: unit:CP - calculated from factors: cd
+  #
+  #  ─── unit:CP ucumCode: [no ucumCode] (correct: cd)
+unit:CP qudt:ucumCode "cd"^^<http://qudt.org/schema/qudt/UCUMcs> .
+
+  # MISSING UCUMCODE: unit:KiloA-PER-K - calculated from factors: kA.K-1
+  #
+  #  ──┬ unit:KiloA-PER-K ucumCode: [no ucumCode] (correct: kA.K-1)
+  #    ├──┬ unit:KiloA ucumCode: kA
+  #    │  └─── unit:A ucumCode: A
+  #    └─── unit:K^-1 ucumCode: K
+unit:KiloA-PER-K qudt:ucumCode "kA.K-1"^^<http://qudt.org/schema/qudt/UCUMcs> .
+
+  # MISSING UCUMCODE: unit:KiloGM-PER-K - calculated from factors: kg.K-1
+  #
+  #  ──┬ unit:KiloGM-PER-K ucumCode: [no ucumCode] (correct: kg.K-1)
+  #    ├──┬ unit:KiloGM ucumCode: kg
+  #    │  └─── unit:GM ucumCode: g
+  #    └─── unit:K^-1 ucumCode: K
+unit:KiloGM-PER-K qudt:ucumCode "kg.K-1"^^<http://qudt.org/schema/qudt/UCUMcs> .
+
+  # MISSING UCUMCODE: unit:LB-PER-DEG_F - calculated from factors: [lb_av].[degF]-1
+  #
+  #  ──┬ unit:LB-PER-DEG_F ucumCode: [no ucumCode] (correct: [lb_av].[degF]-1)
+  #    ├──┬ unit:LB ucumCode: [lb_av]
+  #    │  └──┬ unit:KiloGM ucumCode: kg
+  #    │     └─── unit:GM ucumCode: g
+  #    └──┬ unit:DEG_F^-1 ucumCode: [degF]
+  #       └─── unit:K ucumCode: K
+unit:LB-PER-DEG_F qudt:ucumCode "[lb_av].[degF]-1"^^<http://qudt.org/schema/qudt/UCUMcs> .
+
+  # MISSING UCUMCODE: unit:LB-PER-LB - calculated from factors: [lb_av].[lb_av]-1
+  #
+  #  ──┬ unit:LB-PER-LB ucumCode: [no ucumCode] (correct: [lb_av].[lb_av]-1)
+  #    ├──┬ unit:LB ucumCode: [lb_av]
+  #    │  └──┬ unit:KiloGM ucumCode: kg
+  #    │     └─── unit:GM ucumCode: g
+  #    └──┬ unit:LB^-1 ucumCode: [lb_av]
+  #       └──┬ unit:KiloGM ucumCode: kg
+  #          └─── unit:GM ucumCode: g
+unit:LB-PER-LB qudt:ucumCode "[lb_av].[lb_av]-1"^^<http://qudt.org/schema/qudt/UCUMcs> .
+
+  # MISSING UCUMCODE: unit:LunarMass - calculated from factors: kg
```

Comment:
Can't be right.

Reply:
I weeded out a number of such cases, but apparently, there are still some in there. Let me go over the algorithm again.

Reply:
I'll adapt it such that no ucum code/symbol is generated for a unit without hyphen in the localname.

Reply:
OK, sounds better!

Reply:
done